src/ServerTelemetryChannel/Managed/Shared/Implementation/TelemetryChannelEventSource.cs

```diff
@@ -369,7 +369,7 @@ public void TransmissionSendingFailedWarning(string transmissionId, string excep
         [Event(
             55, 
             Keywords = Keywords.UserActionable,
-            Message = "Access to the local storage was denied (User: {1}). If you want Application Insights SDK to store telemetry locally on disk in case of transient network issues please give the process access either to %LOCALAPPDATA% or to %TEMP% folder. After you gave access to the folder you need to restart the process. Currently monitoring will continue but if telemetry cannot be sent it will be dropped. Error message: {0}.", 
+            Message = "Local storage access has resulted in an error (User: {1}). If you want Application Insights SDK to store telemetry locally on disk in case of transient network issues please give the process access to %LOCALAPPDATA% or %TEMP% folder. After you gave access to the folder you need to restart the process. Currently monitoring will continue but if telemetry cannot be sent it will be dropped. Error message: {0}.", 
```

Comment:
might need to link to an actual documentation instead of this long but in-complete (due to lack of instruction for linux/docker) text.